### DISCUSSION TITLE
color of date range picker by nextUi doesn't appear while selection

### DISCUSSION THREAD

**Discussion by MarouaneCrew** [heroui-inc/heroui#3612] [Help]
![Capture d'écran 2024-08-05 151824](https://github.com/user-attachments/assets/65df2eb0-1548-4821-af03-26261bb4d292)

**Comment by wingkwong** [comment]
please provide a minimal reproducible environment

**Reply by MarouaneCrew** [reply]
yeah sorry i forgot, react laravel

**Reply by wingkwong** [reply]
I mean we need you to provide a sandbox environment to see the issue. We couldn't do much just based on a single screenshot.
